libretro-rs/src/av_info.rs

```diff
@@ -0,0 +1,183 @@
+use core::ffi::*;
+use core::ops::*;
+use libretro_rs_sys::*;
+
+/// Rust interface for [retro_system_av_info].
+#[repr(transparent)]
+#[derive(Debug, Clone)]
+pub struct RetroSystemAVInfo(retro_system_av_info);
+
+impl RetroSystemAVInfo {
```

Comment:
Rename to `RetroSystemAudioVideoInfo`? The `AVInfo` bugs me 😅 because it looks like the `I` is part of the initialism.

Reply:
I'm fine with making it `RetroSystemAudioVideoInfo`, but we could also drop the `Retro` prefix on everything to make the name shorter since we're in "breaking changes" mode right now.

Reply:
I'd love that, actually. I was recently thinking about how it's superfluous since Rust has namespaces, so the 1:1 Rust translation would be `retro::SystemAudioVideoInfo`

Reply:
The FQN would, in this case, be `libretro_rs::SystemAudioVideoInfo`. I really dislike that the actual `libretro` crate name is already in use by an abandoned project and `crates.io` won't let you take over a name once it's used. ☹️